core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -204,12 +204,37 @@ public void testSchemaValidation() throws Exception {
                                                                    ImmutableMap.<String, Object>of(
                                                                      CoreValidator.ID, new CoreValidator()));
     new PythonEvaluator(config).configurePipeline(configurer);
+  }
+
+  @Test
+  public void testSchemaValidation() throws Exception {
+    Schema outputSchema = Schema.recordOf(
+      "smallerSchema",
+      Schema.Field.of("intField", Schema.of(Schema.Type.INT)),
+      Schema.Field.of("longField", Schema.of(Schema.Type.LONG)));
+
+    PythonEvaluator.Config config = new PythonEvaluator.Config(
+      "def transform(x, emitter, context):\n" +
+        " y = {}\n" +
+        " y['intField'] *= 1024\n" +
+        " y['longField'] *= 1024\n" +
+        " emitter.emit(y)",
+      outputSchema.toString());
```

Comment:
I see this is a variation of the above...but is this going to work? If `y` is an empty dict, and then you multiply an element that doesn't exist by 1024, isn't this going to happen?:
```
>>> y = {}
>>> y['intField'] *= 1024
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'intField'
```
as the field doesn't exist in `y`, though it does in `x`.